### DISCUSSION TITLE
umi3配置PageContainer无法显示面包屑

### DISCUSSION THREAD

**Discussion by singgl** [umijs/umi#11450] [General]
import {PageContainer} from '@ant-design/pro-layout';

无法获取到route信息

**Comment by fz6m** [comment]
umi 3 不维护了，建议升级到 umi 4

**Comment by singgl** [comment]
之前是可以的，不知道为啥突然就不行了

**Comment by singgl** [comment]
umi3可以直接升4吗？会不会影响现有项目
